api/src/main/java/org/pmiops/workbench/workspaceadmin/WorkspaceAdminController.java

```diff
@@ -1,150 +1,36 @@
-    response.getReceivedBytes().sort(Comparator.comparing(TimeSeriesPoint::getTimestamp));
-
-    return ResponseEntity.ok(response);
+    return ResponseEntity.ok(workspaceAdminService.getCloudStorageTraffic(workspaceNamespace));
   }
 
   @Override
   @AuthorityRequired({Authority.RESEARCHER_DATA_VIEW})
-  public ResponseEntity<AdminFederatedWorkspaceDetailsResponse> getFederatedWorkspaceDetails(
-      String workspaceNamespace) {
-    Optional<DbWorkspace> workspaceMaybe =
-        workspaceAdminService.getFirstWorkspaceByNamespace(workspaceNamespace);
-    if (workspaceMaybe.isPresent()) {
-      DbWorkspace dbWorkspace = workspaceMaybe.get();
-
-      String workspaceFirecloudName = dbWorkspace.getFirecloudName();
-      List<UserRole> collaborators =
-          workspaceService.getFirecloudUserRoles(workspaceNamespace, workspaceFirecloudName);
-
-      AdminWorkspaceObjectsCounts adminWorkspaceObjects =
-          workspaceAdminService.getAdminWorkspaceObjects(dbWorkspace.getWorkspaceId());
-
-      AdminWorkspaceCloudStorageCounts adminWorkspaceCloudStorageCounts =
-          workspaceAdminService.getAdminWorkspaceCloudStorageCounts(
-              dbWorkspace.getWorkspaceNamespace(), dbWorkspace.getFirecloudName());
-
-      List<org.pmiops.workbench.notebooks.model.ListClusterResponse> fcClusters =
-          leonardoNotebooksClient.listClustersByProjectAsService(workspaceNamespace);
-      List<ListClusterResponse> clusters =
-          fcClusters.stream()
-              .map(
-                  fcCluster ->
-                      new ListClusterResponse()
-                          .clusterName(fcCluster.getClusterName())
-                          .createdDate(fcCluster.getCreatedDate())
-                          .dateAccessed(fcCluster.getDateAccessed())
-                          .googleProject(fcCluster.getGoogleProject())
-                          .labels(fcCluster.getLabels())
-                          .status(ClusterStatus.fromValue(fcCluster.getStatus().toString())))
-              .collect(Collectors.toList());
-
-      AdminWorkspaceResources resources =
-          new AdminWorkspaceResources()
-              .workspaceObjects(adminWorkspaceObjects)
-              .cloudStorage(adminWorkspaceCloudStorageCounts)
-              .clusters(clusters);
-
-      FirecloudWorkspace fcWorkspace =
-          fireCloudService
-              .getWorkspaceAsService(workspaceNamespace, workspaceFirecloudName)
-              .getWorkspace();
-
-      return ResponseEntity.ok(
-          new AdminFederatedWorkspaceDetailsResponse()
-              .workspace(workspaceMapper.toApiWorkspace(dbWorkspace, fcWorkspace))
-              .collaborators(collaborators)
-              .resources(resources));
-    } else {
-      return ResponseEntity.status(HttpStatus.NOT_FOUND).build();
-    }
+  public ResponseEntity<WorkspaceAdminView> getWorkspaceAdminView(String workspaceNamespace) {
```

Comment:
The rename here is because I felt like "federated" wan't really an abstraction, but a function of where the information is fetched from, which is an implementation detail.